Detectors/ITSMFT/common/reconstruction/include/ITSMFTReconstruction/RUDecodeData.h

```diff
@@ -132,6 +142,12 @@ int RUDecodeData::decodeROF(const Mapping& mp, const o2::InteractionRecord ir)
         break; // negative code was returned by decoder: abandon cable data
```

Comment:
If the decoding was abandoned because of some detected error, the subsequent verification necessarily triggers a false positive.

Reply:
To avoid checking the chips that have been abandoned due to errors, now, if there is a mismatch between the bytes and the raw stream indicates the start of a chip, then the code fetches the `ChipPixelData` for the corresponding chip ID and checks whether any error was set for it. If so, the code stops the verification.